### ISSUE TITLE
microphone configure when landscape mode is on

### ISSUE THREAD

**Issue by AndrewR8** [guardianproject/haven#288]
![screenshot_20180423-165832](https://user-images.githubusercontent.com/37798851/39133256-7147721e-471c-11e8-9c00-6c7609bc049d.png)
![screenshot_20180423-165839](https://user-images.githubusercontent.com/37798851/39133258-71742ab6-471c-11e8-9ec9-a16a1db005e7.png)

the diagram does not starts from most right side,while  on portrait mode it does.

**Comment by lukeswitz** [comment]
Yes, you can fix anything you’d like. Also applies to motion sensor calibration view. 

It works when launching activity in either however a proper orientation handler needs to be implemented. Thanks 🙏
